starter-code/main.js

```diff
@@ -1,9 +1,33 @@
+"use strict";
+
+var started = false;
+var seconds = 0;
+var idInterval;
+
+
 // Start/Stop Button
-document.getElementById("btnLeft").addEventListener("click", function () {
+document.getElementById("btnLeft").addEventListener("click", function() {
   console.log("Left button clicked");
+
+  if (!started) {
+    seconds = 0;
```

Comment:
when you click "stop" and again click "start" you put seconds = 0.. means the timer rests. You should put seconds = 0 only when clicking the reset button.